packages/web/src/pages/rewards-page/components/ChallengeRewards/ChallengeRewardsTile.tsx

```diff
@@ -109,13 +109,24 @@ export const ChallengeRewardsTile = ({
           </Text>
         </Box>
         {userChallengesLoading && !haveChallengesLoaded ? (
-          <LoadingSpinner className={wm(styles.loadingRewardsTile)} />
+          <Box
+            h='unit10'
+            w='unit10'
+            css={{
+              position: 'absolute',
+              top: '50%',
+              left: '50%',
+              transform: 'translate(-50%, -50%)'
+            }}
```

Comment:
this seems common?

Reply:
yeah i think we should potentially build this into the loading component since it's used a lot... what you think?